booking-manager.js

```diff
@@ -28,55 +22,26 @@ export class BookingManager {
     return newElement;
   }
 
-  // Method for creating the modal
+  // Method for creating the modal, also used in render method, for challenge-button.
   async createBookingPage(challenge) {
-    // Creating the modal, container is the same all the way through booking
-    const bookingSceneContainer = this.createElement(
-      'div',
-      'bookingSceneContainer__Id',
-      'bookingSceneContainer__class',
-      null
-    );
+    // Creating the modal container. Container is the same through booking process.
+    const bookingSceneContainer = this.createElement('div','bookingSceneContainer__Id','bookingSceneContainer__class',null);
```

Comment:
This change only affects the formatting of the code. Having such changes in a git diff makes it much more difficult for a reviewer to sort out the actually meaningful changes to the code, increasing the mental load of the reviewer, and hence the risk that they miss something important.

It's a good idea at the beginning of any team project to agree on rules for formatting (e.g. quote marks, indentation etc) and configure editors or other automated software to enforce that formatting. That way, no one will ever commit code that doesn't conform with the rules, and no one will have to change formatting as part of their PRs.